test/run

```diff
@@ -9,6 +9,8 @@ if [ -n "$TEST_SCENARIO" ]; then
     test/check-cli TestLiveIso
   elif [ "$TEST_SCENARIO" == "qcow2" ]; then
     test/check-cli TestQcow2
+  elif [ "$TEST_SCENARIO" == "lorax" ]; then
+    test/check-lorax
```

Comment:
How long does this test need to run? I'm hesitant to add too many statuses, because `make vm` is costly and we have to make a PR to cockpit every time we add a scenario. We only split out some of them because they take a really long time to run.

Reply:
It took about 30 minutes in the last test run before I moved it into a new scenario (because atodorov said it would be better there since it takes longer to run).